cli/src/main.rs

```diff
@@ -0,0 +1,41 @@
+use std::str::FromStr;
+
+use clap::{command, Parser, Subcommand};
+
+#[derive(Parser, Debug)]
+#[command(author, version, about, long_about = None)]
+#[command(propagate_version = true)]
+struct CliArguments {
+    #[command(subcommand)]
+    command: Command,
+}
+
+#[derive(Subcommand, Debug)]
+enum Command {
+    TestIssueCredential {
+        #[arg(short, long)]
+        input_file_path: String,
+    },
+    TestIssuePresentation {
+        #[arg(short, long)]
+        input_file_path: String,
+    },
+}
+
+fn main() {
+    let args = CliArguments::parse();
+    match args.command {
+        Command::TestIssueCredential { input_file_path } => {
+            //read the JSON string file in input_file_path and parse it into a VerifiableCredential
+            //print the VerifiableCredential
+            let input = std::fs::read_to_string(input_file_path).unwrap();
```

Comment:
Is it worthwhile to handle errors like this in a nicer way than just panicking (e.g. have the entire `main()` function return some kind of formatted error type)? Who do we see as the ultimate audience for this cli tool?

Reply:
@gshuflin For the foreseeable future this would just be for internal use, I can imagine a future where this could be used in other contexts theoretically but not for a while if at all.

What would be the benefit of handling this more nicely? This CLI would always exit after a command is finished

Reply:
Panicking if you can't do something like open a file passed into a command line app is kind of unfriendly (as opposed to a nicely-formatted error message), but if the only people using this tool for the forseeable future will be us then I probably wouldn't spend that much time trying to clean it up now.

Reply:
Yeah I agree its unfriendly but like you said as an internal tool I don't think its an urgent/valuable enough issue right now